### PR TITLE
fix: pass active count to query context instead of timestamp

### PR DESCRIPTION
**Pull request by zhagnlu**
#29319

### CODE REVIEW

**Review by longjiquan — CHANGES_REQUESTED** [review]
LGTM for the code changes, but there is typo in the pr title (and commits). Maybe you should change `instread` to `instead`.

File: internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp [hunk 1]
```diff
@@ -74,15 +74,19 @@ void
 ExecPlanNodeVisitor::ExecuteExprNodeInternal(
     const std::shared_ptr<milvus::plan::PlanNode>& plannode,
     const milvus::segcore::SegmentInternalInterface* segment,
+    int64_t row_num,
```
Comment (czs007) [inline internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp:77]: The variable "row_num" typically represents the line number. The naming of this variable could be improved in this context.

Comment (zhagnlu) [reply internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp:77]: still using active_count ?

Comment (czs007) [reply internal/core/src/query/visitors/ExecPlanNodeVisitor.cpp:77]: lgtm

**Pull request merged**